CodeStarterCamp_Week2/main.swift

```diff
@@ -8,40 +8,29 @@
 
 import Foundation
 
-//Step1의 순서도를 바탕으로 로또 당첨 번호를 생성하는 함수를 생성합니다.
 func generateLottoNumbers() -> Set<Int> {
     let numbers: Array<Int> = Array(1...45)
     let randomNumbers = numbers.shuffled()
     let lottoNumbers: Set<Int> = Set(randomNumbers[0...5])
     return lottoNumbers
 }
-
 let lottoNumbers = generateLottoNumbers()
 
-//내가 찍은 번호 배열을 myLottoNumbers 변수에 할당합니다.
-let myLottoNumbers: Set<Int> = [5, 13, 23, 25, 37, 42]
+func generateLottoPack(lottoNumbers: Set<Int>) -> Dictionary<String, String> {
+    var lottoNumberDict: Dictionary<String, String> = Dictionary<String, String>()
+    for count in 1...5 {
+        lottoNumberDict["\(count)회차"] = "\(lottoNumbers)"
+    }
+    return lottoNumberDict
+}
+let lottoNumberDict = generateLottoPack(lottoNumbers: lottoNumbers)
 
-//찍은 번호와 로또 당첨 번호 의 겹치는 숫자를 확인하는 함수를 생성합니다.
-func compareLottoNumbers(myLottoNumbers: Set<Int>, with lottoNumbers: Set<Int>) {
-    let intersectSet: Set<Int> = lottoNumbers.intersection(myLottoNumbers)
-    if intersectSet.isEmpty {
-        print("아쉽지만 겹치는 번호가 없습니다.")
+func findSecondLottoNumber(week: String, lottoNumberDict: Dictionary<String, String>) {
+    if let winnigLottoNumber = lottoNumberDict["\(week)"] {
+        let finalLottoNumber = "\(winnigLottoNumber)".trimmingCharacters(in: ["[","]"])
```

Comment:
```suggestion
        let finalLottoNumber = winnigLottoNumber.trimmingCharacters(in: ["[","]"])
```
[String: String] 타입의 Dictionary 인 `lottoNumberDict` 에서 가져온 값 `winnigLottoNumber`는 이미 String 타입이라 따로 문자열 보간법을 사용해 String 변환을 해주지 않아도 될 것 같아요!

Reply:
코드를 `Dictionary<String, String>`에서 `[String: Set<Int>]`로 변경하여 
`let finalLottoNumber = "\(winnigLottoNumber)".trimmingCharacters(in: ["[","]"])`
이건 유지하겠습니다!